<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>631</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Subject</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>438</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Grade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Credits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>GPA</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>625</width>
            <height>386</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Result:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Calculate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Subject">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionCalculate"/>
    <addaction name="separator"/>
    <addaction name="actionCalculate_2"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuNew_Subject"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCalculate"/>
   <addaction name="separator"/>
   <addaction name="actionCalculate_2"/>
   <addaction name="separator"/>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCalculate">
   <property name="icon">
    <iconset resource="../debug/resources.qrc">
     <normaloff>:/img/add_list1600.png</normaloff>:/img/add_list1600.png</iconset>
   </property>
   <property name="text">
    <string>New Subject</string>
   </property>
  </action>
  <action name="actionCalculate_2">
   <property name="icon">
    <iconset resource="../debug/resources.qrc">
     <normaloff>:/img/15-512.png</normaloff>:/img/15-512.png</iconset>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="../debug/resources.qrc">
     <normaloff>:/img/player-08-128.png</normaloff>:/img/player-08-128.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../debug/resources.qrc">
     <normaloff>:/img/close_4-512.png</normaloff>:/img/close_4-512.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../debug/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>label_6</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
